INFO     root:conftest.py:124 Starting test: test_basic_auth[--Not authorized\n].
INFO     WDM:logger.py:11 xdist worker is: gw1
INFO     WDM:logger.py:11 ====== WebDriver manager ======
INFO     WDM:logger.py:11 Get LATEST chromedriver version for google-chrome
INFO     WDM:logger.py:11 Get LATEST chromedriver version for google-chrome
INFO     WDM:logger.py:11 Driver [/home/runner/.wdm/gw1/drivers/chromedriver/linux64/141.0.7390.122/chromedriver-linux64/chromedriver] found in cache
INFO     root:conftest.py:142 Started recording: tests_recordings/gw1/test_basic_auth[--Not authorized\n]_20251028_093242/test_basic_auth[--Not authorized\n]_20251028_093242.mp4
INFO     root:base_page.py:76 Navigating to URL: https://the-internet.herokuapp.com/.
INFO     root:base_page.py:78 Navigation completed.
INFO     root:base_page.py:57 Waiting for page to load with indicator ('css selector', 'h1.heading') for 10s.
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
INFO     root:base_page.py:62 Page loaded successfully with indicator ('css selector', 'h1.heading').
INFO     root:basic_auth_page.py:17 .
INFO     root:basic_auth_page.py:25 .
INFO     root:base_page.py:76 Navigating to URL: http://the-internet.herokuapp.com/basic_auth.
INFO     root:base_page.py:78 Navigation completed.
INFO     root:basic_auth_page.py:30 Attempting to get authorization message.
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
WARNING  urllib3.connectionpool:connectionpool.py:327 Connection pool is full, discarding connection: localhost. Connection pool size: 1
ERROR    root:base_page.py:32 Timeout after 10s for locator ('css selector', 'div#content p'): Message: 
Stacktrace:
#0 0x56250fcb6fea <unknown>
#1 0x56250f734d06 <unknown>
#2 0x56250f786e6d <unknown>
#3 0x56250f787101 <unknown>
#4 0x56250f7d56b4 <unknown>
#5 0x56250f7ace3d <unknown>
#6 0x56250f7d2b3b <unknown>
#7 0x56250f7acbe3 <unknown>
#8 0x56250f7792d2 <unknown>
#9 0x56250f779f91 <unknown>
#10 0x56250fc7a9e8 <unknown>
#11 0x56250fc7e84f <unknown>
#12 0x56250fc61ec9 <unknown>
#13 0x56250fc7f3f5 <unknown>
#14 0x56250fc4774f <unknown>
#15 0x56250fca3cc8 <unknown>
#16 0x56250fca3ea3 <unknown>
#17 0x56250fcb5f83 <unknown>
#18 0x7f014ca9caa4 <unknown>
#19 0x7f014cb29c6c <unknown>
INFO     root:conftest.py:146 Stopping video recording...
INFO     utils.video_recorder:video_recorder.py:95 Video created: tests_recordings/gw1/test_basic_auth[--Not authorized\n]_20251028_093242/test_basic_auth[--Not authorized\n]_20251028_093242.mp4
INFO     root:conftest.py:102 Quitting driver for browser: chrome.